docs(faq): add pro-rata billing question (#2529)
scaleway/docs-content: faq/billing.mdx

<Concept>

## What does pro-rata billing for Dedibox and Web Hosting services mean?

> In the initial month of a subscription, you are billed pro-rata in the following month. This means the charge is adjusted to reflect the portion of the first month you used the service.

Reviewer: ```suggestion
You care billed pro-rata for the initial month of a subscription. This means the charge is adjusted to reflect the portion of the first month you used the service.
```

Reviewer: I think it's confusing to point out the bill arrives in the following month, maybe we can just eliminate the mention of when the pro rata bill for the first month actually arrives?

> In the initial month of a subscription, you are billed pro-rata in the following month. This means the charge is adjusted to reflect the portion of the first month you used the service.

Reviewer 2: ```suggestion
After you have completed your first month of subscription, you are billed on a pro-rata basis the following month. This means that the charge is adjusted to reflect your usage of the service on the first month.
```
